lib/routes/profile.js

```diff
@@ -18,6 +19,20 @@ function hasProfileScope(scopes) {
   return false;
 }
 
+function computeEtag(profile) {
+  if (profile) {
+    var key = profile.email;
+    if (profile.avatar && profile.avatar.id) {
+      key += profile.avatar.id;
+    }
+    if (profile.displayName) {
+      key += profile.displayName;
+    }
```

Comment:
To prevent additional fields from being forgotten, how about just checksuming the JSON? basically `return checksum(JSON.stringify(profile))`